### ISSUE TITLE
[BUG] 'Builder' object has no attribute 'generate_runtime'

### ISSUE THREAD

**Issue by VeNoMouS** [dashingsoft/pyarmor#1249]
```
root@tvz0r:/Project/Helheim# pyarmor gen runtime -O dist/helheim/
INFO     Python 3.9.2
INFO     Pyarmor 8.2.0 (pro), 005005, helheim
INFO     Platform linux.x86_64
INFO     start to generate runtime package
ERROR    'Builder' object has no attribute 'generate_runtime'
root@tvz0r:/Project/Helheim#
```

Another bug, when using bcc and you have platform arg defined, it doesnt check that is what the OS is..
```
root@tvz0r:/Project/Helheim# pyarmor gen -O dist/helheim/ --platform linux.x86_64  --enable-bcc --enable-jit  helheim
INFO     Python 3.9.2
INFO     Pyarmor 8.2.0 (pro), 005005, helheim
INFO     Platform linux.x86_64
INFO     use runtime platforms: ['linux.x86_64']
ERROR    bcc mode does not support cross platform
root@tvz0r:/Project/Helheim#
```

**Comment by jondy** [comment]
`gen runtime` has not released and not documented, it'll be released in future version.

The second issue will be fixed in next time, now bcc raises error once option `--platform` is used.

**Comment by jondy** [comment]
Fixed in 8.2.1
